views/house.erb

```diff
@@ -0,0 +1,58 @@
+<div class="page-section page-section--grey text-center">
+  <div class="container">
+    <h3>We’re cataloguing every politician in <%= @page.country.name %>.</h3>
+    <p>We currently have information for…</p>
+  </div>
```

Comment:
This is a little misleading now, because it implies that this is _all_ we have for this country.